programmers/제일-작은-수-제거하기.js

```diff
@@ -0,0 +1,13 @@
+function solution(arr) {
+  const answer = [];
+
+  arr.forEach((el) => {
+    if (el !== Math.min(...arr)) {
+      answer.push(el);
+    }
+  });
+  if (arr.length === 1) {
+    answer.push(-1);
+  }
+  return answer;
+}
```

Comment:
```suggestion
function solution(arr) {
   return (arr.length === 1) ? [-1] : arr.filter((number) => {return number !== Math.min(...arr)})
}
```
filter 메서드가 바로 이럴 때 힘을 발휘합니다.
for문 돌려서 찾는거보다 filter로 조건에 해당하는 것들만 반환하면 훨씬 편해요